packages/theming/atlas/tests/e2e/allScreens.spec.ts

```diff
@@ -0,0 +1,44 @@
+const fs = require("fs");
+const { join, resolve } = require("path");
+const cwd = process.cwd();
+const xmlParser = require("fast-xml-parser");
+
+// For every module mx exports a folder, and in that folder we have bunch of xml's
+// which might contains URL field
+const individualPageFolders = join(cwd, "tests/testProject/deployment/web/pages/en_US");
+
+// Make an async function that gets executed immediately
+function* getFiles(dir) {
+    const dirents = fs.readdirSync(dir, { withFileTypes: true });
+    for (const dirent of dirents) {
+        const res = resolve(dir, dirent.name);
+        if (dirent.isDirectory()) {
+            yield* getFiles(res);
+        } else {
+            yield res;
+        }
+    }
+}
+
+describe("Screenshots of the pages for", () => {
+    beforeAll(() => {
+        browser.url("/");
+    });
+
+    for (const f of getFiles(individualPageFolders)) {
+        const file = fs.readFileSync(f, { encoding: "utf8" });
+        if (xmlParser.validate(file) === true) {
+            // optional (it'll return an object in case it's not valid)
```

Comment:
I reckon this comment should be on the same line as it applies (line 30, based on reading [these docs](https://github.com/NaturalIntelligence/fast-xml-parser#xml-to-json))

Reply:
Removed the comment completely